### PR TITLE
Load more on aggregation

### PR DESCRIPTION
**Pull request by LHBruneton-C2C**
Add action and effect to load more on one aggregation
Output from dumb component
Dispatch action from container

### CODE REVIEW

File: libs/search/src/lib/state/reducer.ts [hunk 1]
```diff
@@ -127,6 +127,18 @@ export function reducer(
         },
       }
     }
+    case fromActions.REQUEST_MORE_ON_AGGREGATION: {
+      const clone = JSON.parse(JSON.stringify(state.config.aggregations))
+      clone[action.key].terms.size += 20
```
Comment (fgravin) [inline libs/search/src/lib/state/reducer.ts:132]: no logic in reducers, `20` should be in the payload.

**Review by fgravin — CHANGES_REQUESTED** [review]
Looks great except one thing to change in the new Action.
Can't test it cause the api has changed on titellus.

**Comment by fgravin** [comment]
Great @LHBruneton-C2C, since the rebase, load more action set the `response.aggs` instead of patching it.
Also, see if you can just load the extra items instead of loading them all, using the `from` and size` `search options.
Thanks

**Comment by LHBruneton-C2C** [comment]
On rebasing, I saw that there was a dependency from lib-ui to lib-search, in facet-block component, `import { AggregationsTypesEnum } from '../../../../../search/src/lib/facets/facets.model'` => is it OK ?

**Comment by fgravin** [comment]
> On rebasing, I saw that there was a dependency from lib-ui to lib-search, in facet-block component, `import { AggregationsTypesEnum } from '../../../../../search/src/lib/facets/facets.model'` => is it OK ?

not it's not
model should be shared if needed

**Review by fgravin — APPROVED** [review]
Excellent thanks

**Pull request merged**